merlin/systems/dag/op_runner.py

```diff
@@ -50,8 +52,14 @@ def __init__(
 
     def execute(self, tensors):
         """Run transform on multiple operators"""
+
+        selector = ColumnSelector("*")
```

Comment:
Systems operators have not historically paid very close attention to column selectors, and have defaulted to processing all input columns (except in a few particular cases like `UnrollFeatures`.) We've tried to align the Systems operator API more closely with the NVTabular operator API so that we can use `LocalExecutor`, but that API change also affects older code like this class that's used with Triton ensembles. Here we just supply a default selector that selects everything, which mirrors the existing behavior with the new API.